### ISSUE TITLE
Hardcoded path to mods_to_dc.xsl is problematic

### ISSUE THREAD

**Issue by t4k** [SFULibrary/islandora_datastream_crud#40]
Related to #17 maybe.

https://github.com/mjordan/islandora_datastream_crud/blob/3a915989e78199481018a5cc04a8e9fdab076ab7/includes/utilities.inc#L366-L369

We don’t have our site root in `/var/www/drupal/` and we are not using any kind of custom transform, so it would be nice if we could be guaranteed to pick up the standard one.

I can create a pull request, but the change should be easy enough:

```php
  $xslt_path = drush_get_option(
    'dc_transform',
    drupal_get_path('module', 'xml_forms') . '/builder/transforms/mods_to_dc.xsl'
  );
```

Let me know if I should make a PR for this.

**Comment by mjordan** [comment]
@t4k yes please, open a PR. Fallout from #17.

**Comment by mjordan** [comment]
I'll update the README to indicate that the default for `--dc_transform` is the path to the XSLT provided by xml_forms.

**Comment by mjordan** [comment]
Closed with 1d6ea3a39cd4696d84b9f9e85268f99d7b8bb89a. README updated with 664f196a38e32fbe40a97625eee51114747a71a8.
